public/app/plugins/panel/gettingstarted/GettingStarted.tsx

```diff
@@ -201,6 +201,10 @@ const getStyles = stylesFactory(() => {
       justify-content: center;
       align-items: center;
       height: 100%;
+      margin: 0 auto;
+    `,
+    item: css`
+      grid-column: 4 / span 1;
```

Comment:
Is this leftover from something else?

Reply:
Yes, I updated  now :D

Reply:
Thank you for feedback and suggestions, I made the updates and pushed the component :)